.golangci.yml

```diff
@@ -10,7 +10,7 @@ linters:
     - bodyclose
     - depguard
     - dogsled
-    - dupl
+    # - dupl
```

Comment:
Why did you disable this linter ? 
Please remove it from config or add comment with reason

Reply:
OK, i disabled it. There was some code duplication with dbaas resources, not related to this PR.